CRM/MembershipExtras/Form/PaymentPlanSettings.php

```diff
@@ -24,12 +26,28 @@ public function buildQuickForm() {
 
     $settingFields  = $this->getSettingFields();
     $settingFieldNames = [];
+
     foreach ($settingFields  as $name => $field) {
+      $attributes = '';
+
+      switch (true) {
```

Comment:
I always see that such kind of switch ins unhealthy to have in code and does not make any sense, it is better to be replace by if statements in this case, or probably it is better now to remove the whole foreach loop and adding fields one by one since with this new field it does not make much sense to still using the loop.